holidayextras/make-up
# Enable integrations

#### What does this PR do? (please provide any background)

We would like to have more than just the linting checks behind makeup, dependency checking and CSS checks should be arriving soon.

In order to support more checks this work refactors how the current lint checking is called.

I'll be bumping versions in a separate PR.

#### What tests does this PR have?

Unit tests for changed functionality as well as some more tests to increase the overall coverage of this project.

#### How can this be tested?

Running the following will ensure the code still works as before:

    npm run lint

#### Screenshots / Screencast


#### What gif best describes how you feel about this work?

![giphy](https://cloud.githubusercontent.com/assets/3158640/11363114/add4840e-928f-11e5-8aee-05c1127abe2e.gif)

---


#### Reviewers

**Review 1**

**Review 2** \*

**Review 3** _(optional)_

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

\*  for HX this review must be completed by an SE, SA or Project Guru
